mip/model.py

```diff
@@ -1427,8 +1427,7 @@ def remove(
         else:
             raise TypeError(
                 "Cannot handle removal of object of type "
-                + type(objects)
-                + " from model."
+                "{} from model".format(type(objects))
```

Comment:
Would you please add your initial code snippet that causes this error as unittest please?

Reply:
So I wanted to cover most of the `remove` functionality in my test and I stumbled upon another bug:
```
>>> m = Model()
>>> x = m.add_var("x")
>>> m.remove(x)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/markus/OpenSourceDev/python-mip/mip/model.py", line 1424, in remove
    self.vars.remove(vlist)
  File "/home/markus/OpenSourceDev/python-mip/mip/lists.py", line 72, in remove
    v.idx = -1
AttributeError: can't set attribute 'idx'

Reply:
Pls, add a different PR for this. Try to keep things as small as possible.

Reply:
Sure, then this PR is ready imo. On a somewhat unrelated note: I think this project should really start using pre-commit, considering that CI seems to use black and flake8